website/src/content/blog/2025-03-31-typespec-1-0-release/typespec_1_0_release.md

```diff
@@ -0,0 +1,215 @@
+>
+> — _MS Learn Team_
+
+> Before the introduction of TypeSpec and our move to an API first approach, an OAS [OpenAPI Specification] was a byproduct of the implementation phase. Manual OAS reviews were happening at the end of the development cycle and changes were contentious because of the code rework implied and the likely impact to the release schedule. TypeSpec provides a way to embed [our] guidelines and rules directly in the design. Reviews are now focused more on the design than on the rules and requires less time." [(read more)](../2024-11-04-typespec-at-lseg/blog.md)
+>
+> — _London Stock Exchange Group_
+
+You can explore more real-world TypeSpec usage on our [videos page](https://typespec.io/videos/), featuring presentations from both Microsoft teams and external community members showing how they're using TypeSpec to improve their API development workflows.
+
+## What's in the 1.0 Release Candidate
+
+The 1.0-RC includes a foundation of stable core components alongside preview features that are still actively being developed based on community feedback.
+
+### Stable components
+
+These components have been thoroughly tested and are considered ready for testing in your production environment:
+
+- **Compiler and core libraries**
+
+  - [@typespec/compiler](https://www.npmjs.com/package/@typespec/compiler): The TypeSpec language compiler
+  - [@typespec/http](https://www.npmjs.com/package/@typespec/http): HTTP protocol support
+  - [@typespec/openapi](https://www.npmjs.com/package/@typespec/openapi): OpenAPI support
+
+- **IDE support**
+
+  - [typespec-vscode](https://marketplace.visualstudio.com/items?itemName=typespec.typespec-vscode): VS Code extension
+  - [typespec.typespecvs](https://marketplace.visualstudio.com/items?itemName=typespec.typespecvs): Visual Studio extension
+
+- **Stable emitters**
+  - [@typespec/openapi3](https://www.npmjs.com/package/@typespec/openapi3): OpenAPI 3.0 output
+  - [@typespec/json-schema](https://www.npmjs.com/package/@typespec/json-schema): JSON Schema output
+
+### Preview features
+
+We're also including several components in preview state that are still being developed. We welcome your feedback on these features:
+
+- **Protocol emitters**
+
+  - [@typespec/protobuf](https://www.npmjs.com/package/@typespec/protobuf): Protocol Buffer definitions
+
+- **Libraries**
+
+  - [@typespec/events](https://www.npmjs.com/package/@typespec/events)
+  - [@typespec/rest](https://www.npmjs.com/package/@typespec/rest)
+  - [@typespec/sse](https://www.npmjs.com/package/@typespec/sse)
+  - [@typespec/streams](https://www.npmjs.com/package/@typespec/streams)
+  - [@typespec/versioning](https://www.npmjs.com/package/@typespec/versioning)
+  - [@typespec/xml](https://www.npmjs.com/package/@typespec/xml)
+
+- **Client/server code generation (Preview)**
```

Comment:
Is there a reason we call out (Preview) here but didn't in the other categories under Preview features?

Reply:
Good catch, thanks. That was a holdover from a previous draft.